app/models/manageiq/providers/redhat/infra_manager/inventory.rb

```diff
@@ -1,3 +1,133 @@
-module ManageIQ::Providers::Redhat::InfraManager::Inventory
-  require_nested :Builder
```

Comment:
I don't think we can get rid of the `Inventory::Builder` it is still part of the "graph refresh api"

Reply:
I thought the require it was for the Builder tat was implementing the strategy pattern with the choice between v3 and v4.

Reply:
Hi, I've checked again and I think the "require_nested :Builder" is leading to https://github.com/ManageIQ/manageiq-providers-ovirt/blob/master/app/models/manageiq/providers/redhat/infra_manager/inventory/builder.rb
so according to me it was to remove due to the fact it was never called again. The collector was using directly the V4 inside strategies. 
https://github.com/ManageIQ/manageiq-providers-ovirt/blob/dfdd968d88940977db7a553a0c5b1b437007fc68/app/models/manageiq/providers/redhat/inventory/collector/infra_manager.rb#L4
hope not to be wrong.

Reply:
Ah you are right, this is `infra_manager/inventory` not `inventory/`

Reply:
thx a lot @agrare for your time.